packages/core/test/tracing/clsHooked/emitter-listener_test.js

```diff
@@ -129,7 +129,7 @@ describe('emitter-listener', () => {
     expect(ee.removeListener.__wrapped, 'removeListener is not wrapped').to.not.exist;
 
     expect(function () {
-      ee.__unwrap();
+      ee.__unwrapIn();
```

Comment:
Not sure if `In` is supposed to mean "in something" or an abbreviation of "Instana".
If that is the case, I would suggest that we actually write "Instana" to avoid interpretation of "in something".
If it really means "in", what exactly do you mean by that? In what?

Reply:
Good point, it was actually meant as an abbreviation of Instana, but I see how that is confusing. I'll change it accordingly.